options.js

```diff
@@ -1,26 +1,97 @@
 ((browser) => {
-  document.querySelector('h1').textContent =
-    browser.i18n.getMessage('link_copy_style');
+  document.querySelector('h1').textContent = browser.i18n.getMessage('link_copy_style');
   document.querySelector('.rich').textContent = browser.i18n.getMessage('rich');
   document.querySelector('.raw').textContent = browser.i18n.getMessage('raw');
+  document.querySelector('.rich_plus_raw').textContent = browser.i18n.getMessage('rich_plus_raw');
+  document.querySelector('h4').textContent = browser.i18n.getMessage('link_text');
+  document.querySelector('.link_text_option_1').textContent = browser.i18n.getMessage('link_text_option_1');
+  document.querySelector('.link_text_option_2').textContent = browser.i18n.getMessage('link_text_option_2');
+  document.querySelector('.link_text_option_3').textContent = browser.i18n.getMessage('link_text_option_3');
+  document.querySelector('.link_text_option_4').textContent = browser.i18n.getMessage('link_text_option_4');
 
   const rich = document.querySelector('#rich');
   const raw = document.querySelector('#raw');
+  const rich_plus_raw = document.querySelector('#rich_plus_raw');
+
+  const link_text_option_1 = document.querySelector('#link_text_option_1');
+  const link_text_option_2 = document.querySelector('#link_text_option_2');
+  const link_text_option_3 = document.querySelector('#link_text_option_3');
+  const link_text_option_4 = document.querySelector('#link_text_option_4');
+  const link_text_input = document.querySelector('#link_text_input');
+
+  const enableLinkTextOptions = (enabled) => {
+    document.getElementsByName("link_text").forEach((e) => {
+      e.disabled = !enabled;
+    });
+  };
 
   // Restore previous settings.
-  browser.storage.sync.get({linkStyle: 'rich'}, (items) => {
+  browser.storage.sync.get({linkStyle: 'rich', linkText: document.querySelector('.link_text_option_1').textContent}, (items) => {
     rich.checked = items.linkStyle === 'rich';
     raw.checked = items.linkStyle === 'raw';
+    rich_plus_raw.checked = items.linkStyle === 'rich_plus_raw';
+
+    link_text_option_1.checked = items.linkText === document.querySelector('.link_text_option_1').textContent;
+    link_text_option_2.checked = items.linkText === document.querySelector('.link_text_option_2').textContent;
+    link_text_option_3.checked = items.linkText === document.querySelector('.link_text_option_3').textContent;
+    if (!link_text_option_1.checked && !link_text_option_2.checked && !link_text_option_3.checked) {
+        link_text_option_4.checked = true;
+        link_text_input.value = items.linkText;
+    }
+
+    enableLinkTextOptions(rich_plus_raw.checked);
```

Comment:
Here and throughout: please use `lowerCamelCase` variable names as per the [Google JS style guide](https://google.github.io/styleguide/jsguide.html#naming-local-variable-names). Thanks!

Reply:
Done in af62d525855da94d9bbc8b0bb3a751c290660d4d